src/Connection/Cluster/RedisCluster.php

```diff
@@ -227,6 +229,9 @@ private function queryClusterNodeForSlotMap(NodeConnectionInterface $connection)
                 }
 
                 ++$retries;
+                
+                sleep(2**$retries);
```

Comment:
Sleeping for 2s seems aggressive, can we make this more granular.

Reply:
My last change introduce exponential backoff retries and $minRetryAfter